cmd/contact-exporter/main.go

```diff
@@ -0,0 +1,122 @@
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+
+	"gopkg.in/gorp.v1"
+
+	"github.com/jmhodges/clock"
+	"github.com/letsencrypt/boulder/cmd"
+	blog "github.com/letsencrypt/boulder/log"
+	"github.com/letsencrypt/boulder/sa"
+)
+
+type contactExporter struct {
+	log   blog.Logger
+	dbMap *gorp.DbMap
+	clk   clock.Clock
+}
+
+func (c contactExporter) findContacts() ([]string, error) {
+	var contactsJSON []string
+	var contacts []string
+
+	_, err := c.dbMap.Select(
+		&contactsJSON,
+		`
+			SELECT DISTINCT (contact) AS active_contact
```

Comment:
Generally we've avoiding using DISTINCT because we can de-duplicate in Boulder memory instead of MySQL memory, slightly reducing load on the MySQL instances. Also, since we'll have to de-dupe after JSON parsing anyhow, this is redundant.